src/features/auth/services/api.ts

```diff
@@ -17,7 +17,9 @@ export const authService = {
     });
   },
   logout: async () => {
-    return api.private.post('/auth/logout');
+    return api.bff.client.post('/api/v1/auth/logout', {
+      withCredentials: true,
+    });
```

Comment:
clientToBffHttp는 이미 기본적으로 크리덴셜을 포함하도록 설정되어 있으므로, logout 함수에서 다시 { withCredentials: true }를 요청 Config 객체로 전달하는 것은 불필요한 중복중복입니다

```suggestion
logout: async () => {
  return api.bff.client.post('/api/v1/auth/logout');
},
```